charmcraft/commands/store/__init__.py

```diff
@@ -583,3 +608,54 @@ def _get_lib_info(*, full_name=None, lib_path=None):
     return LibData(
         lib_id=libid, api=libapi, patch=libpatch, content_hash=content_hash, content=content,
         full_name=full_name, path=lib_path, lib_name=lib_name, charm_name=charm_name)
+
+
+class CreateLibCommand(BaseCommand):
+    """Create a charm library."""
+    name = 'create-lib'
+    help_msg = "Create a charm library."
+    overview = textwrap.dedent("""
+        Create a charm library.
+
+        It will request a unique ID from Charmhub and bootstrap a
+        template file in the proper local directory.
+
+        It will automatically take you through the login process if
+        your credentials are missing or too old.
+    """)
+
+    def fill_parser(self, parser):
+        """Add own parameters to the general parser."""
+        parser.add_argument(
+            'lib_name', metavar='lib-name',
```

Comment:
This is because the `list-lib` command also needs a name, but the charm name! So I used `--charm-name` there, and `--lib-name` here. But if you think it's enough difference to use `--charm-name` there, and when just using `--name` is obviously the library name, I'm fine to change it.

Reply:
*`<Chipaca> for create-lib's argument, 'name' makes the most sense, as what else would it be`*

Changing it.